diesel_tests/tests/expressions/ops.rs

```diff
@@ -91,6 +91,6 @@ fn mix_and_match_all_numeric_ops() {
         (3, 'Jim'), (4, 'Bob')").unwrap();
 
     let expected_data = vec![4, 6, 7, 9];
-    let data = users.select(id * 3 / 2 + 4 - 1).load(&connection);
+    let data = users.select(id * 3 / 2 + 4 - 1).order(id).load(&connection);
```

Comment:
We have a bunch of intermittent failures from needing an order added, would definitely prefer in a separate PR. It affects most of our tests. Actually making me contemplate default ordering.